app/controllers/v1/profiles_controller.rb

```diff
@@ -63,6 +63,12 @@ def tailoring_file
 
     private
 
+    def update_relationships
+      add_inventory_hosts(new_host_ids || [])
+      profile.update_hosts(new_host_ids)
+      profile.update_rules(ids: new_rule_ids)
+    end
+
```

Comment:
This new method is because of rubocop, but also the logic is identical for `create` and `update`, so it makes sense to consolidate them.